go.mod

```diff
@@ -89,7 +89,7 @@ require (
 	github.com/DataDog/datadog-go/v5 v5.4.0
 	// do not update datadog-operator to 1.2.1 because the indirect dependency github.com/DataDog/datadog-api-client-go/v2 v2.15.0 is trigger a huge Go heap memory increase.
 	github.com/DataDog/datadog-operator v1.1.0
-	github.com/DataDog/ebpf-manager v0.3.9
+	github.com/DataDog/ebpf-manager v0.3.10-0.20231215225001-fcc8d3a8d17f
```

Comment:
we should merge this PR only after merging the PRs in ebpf-manager and releasing a new version

Reply:
Merge on ebpf-manager was pending, I just bumped the version now to 0.4.0.